UPGRADE-2.8.md

```diff
@@ -4,6 +4,42 @@
 Form
 ----
 
+ * The "checkbox" form-type now interprets '0' and '' values as unchecked unless you 
+   explicitly specified one of those as checked-value (the "value"-option). 
+   Any other value will transform to NULL.
+   
+   Before:
+   ```
+   true  <---> '1' (default checked-value)
+   false <---> null
+   (any other value were true as well)
+   true  <--- ''
+   true  <--- '0'
+   true  <--- 'foobar'
+   ```
+   
+   After:
+   ```
+   true  <---> '1' (default checked-value)
+   false <---> null
+   true  <---> true
```

Comment:
This line and the line below should be a `<---`. `true` (model data) is transformed to `1` by default, as shown above.